addons/supplies/XEH_postInitClient.sqf

```diff
@@ -10,3 +10,38 @@ if (!hasInterface) exitWith {};
 }] call CBA_fnc_addEventHandler;
 
 [QGVAR(loadTraits), ace_player] call CBA_fnc_serverEvent;
+
+// Lootbox functionality
+private _lootBoxClasses = [
+    "tacgt_Lootbox_Handgun",
+    "tacgt_Lootbox_SMG",
+    "tacgt_Lootbox_Rifle",
+    "tacgt_Lootbox_GL",
+    "tacgt_Lootbox_LMG",
+    "tacgt_Lootbox_Marksman"
+];
+
+{
+    [
+        _x,
```

Comment:
Create a base class here in Mods which GTArmory can then inherit from. You can only add it to the base class and context menu will apply for inherited items as well. I recommend `GVAR(randomizedBox)` or anything over "lootbox".

Also change the config entry to `GVAR(possibleItems)` or something similar.

Reply:
@TyroneMF This sounds like a you thing